### PR TITLE
pkp/pkp-lib#1050 Add review form responses to reviews report

### PR DESCRIPTION
**Pull request by mfelczak**
Added. Also includes bug fix for declined and cancelled columns, which included incorrect values.

### CODE REVIEW

File: plugins/reports/reviews/ReviewReportPlugin.inc.php [hunk 1]
```diff
@@ -137,6 +140,50 @@ function display(&$args) {
...
 		fclose($fp);
 	}
+
+	/**
+	 * Retrieve and parse review form questions and responses for a review
+	 * @param $reviewId int
+	 * @param $reviewFormId int
+	 * @return string
+	 */
+	function _getReviewFormResponses($reviewId, $reviewFormId) {
```
Comment (asmecher) [inline plugins/reports/reviews/ReviewReportPlugin.inc.php]: This is borrowed from the "import peer reviews" button, correct? Maybe there's a place to put this where both areas could use it?

Comment (mfelczak) [inline plugins/reports/reviews/ReviewReportPlugin.inc.php]: Good idea. I'll have a look, but I think that code has email-specific text formatting throughout.

**Comment by mfelczak** [comment]
Removed duplicate code and refactored for re-use.

**Comment by mfelczak** [comment]
@asmecher, can you please review the updated PR?

**Comment by asmecher** [comment]
That looks good, though I haven't tested it. Have you tried both use cases? If so, I'll merge, and then can you forward-port to ojs-dev-2_4?

**Comment by mfelczak** [comment]
Yes, tested with both use cases.

**Comment by asmecher** [comment]
Great, thanks!

**Pull request merged**